pixi/README.md

```diff
@@ -0,0 +1,54 @@
+# CUDA accelerated Pixi environment
+
+> [!NOTE]
+> While this approach does work, as there is no shared file system cache that Pixi can leverage it is less efficient than building a Linux container with the Pixi environment and using a `container` universe job.
```

Comment:
The efficiency statement is with regards to not having to redownload multiple GBs of packages on every worker from conda-forge as having all of them pull a container image that should be cached on CHTC's systems should be a lower bandwidth burden and faster. Though I don't actually know how to measure the amount of time it takes to pull a cached container image on CHTC, where timing an install is easy.